cypress/integration/lsp-tokens/1-mint-and-redeem.js

```diff
@@ -0,0 +1,81 @@
+/* eslint-disable */
```

Comment:
Why disable eslint in these files?

Reply:
The linter is too strict for what we're trying to do here. For instance, it won't allow require statements in the hardhat scripts, in spite of being commonjs.

Basically, I'd have to dig in and modify the linter to be more permissive, and that seems against what I wanna do in these tests anyway.

Reply:
I see. For eslint you can apply different rules for different extensions, so we could (theoretically) name all these files `.cjs`, which is typical for commonjs files in a package that uses import syntax. Then we could just tell the linter to use commonjs rules for those files.

Not a huge deal, though.